### PR TITLE
Add repository_branch to client config

### PR DESCRIPTION
**Pull request by ryansch**
The server side branch functionality is already on master.  This makes it possible to use outside of bench testing.

### CODE REVIEW

File: lib/shipitron/client/run_ecs_tasks.rb [hunk 1]
```diff
@@ -125,6 +126,10 @@ def command_args(cluster)
             ary.concat(context.ecs_service_templates.map {|t| Base64.urlsafe_encode64(t)})
           end
 
+          unless context.repository_branch.nil?
+            ary.concat ['--repository-branch', escaped(:repository_branch)]
```
Comment (morganick) [inline lib/shipitron/client/run_ecs_tasks.rb:130]: What does this eq? `...-- repository-branchescapedbranch...`

Comment (ryansch) [reply lib/shipitron/client/run_ecs_tasks.rb:130]: Huh?

Comment (morganick) [inline lib/shipitron/client/run_ecs_tasks.rb:130]: Nm, I'm dumb. Go for it.

**Comment by morganick** [comment]
🚀

**Comment by gjamerson** [comment]
🚢

**Pull request merged**